embrace-android-api/src/main/kotlin/io/embrace/android/embracesdk/internal/api/UserApi.kt

```diff
@@ -10,28 +10,32 @@ public interface UserApi {
 
     /**
      * Sets the user ID. This would typically be some form of unique identifier such as a UUID or database key for the user.
-     * This ID will persist across app launches until it is explicitly removed using [.clearUserIdentifier]
+     * This ID will persist across app launches until it is explicitly removed using [clearUserIdentifier]
      * or otherwise cleared.
      *
      * @param userId the unique identifier for the user
      */
+    @Deprecated("Discourage storing personal identifying information in telemetry")
     public fun setUserIdentifier(userId: String?)
 
     /**
      * Clears the currently set user ID. For example, if the user logs out.
      */
+    @Deprecated("Discourage storing personal identifying information in telemetry")
     public fun clearUserIdentifier()
```

Comment:
IMO we should keep these but document that their value shouldn't be set to sensitive info like an email etc. If it's just a UUID then there's not much difference between what we generate for UUID, but I can see this being useful if folks want to link data with other systems. Email/name should get deprecated for sure though

Reply:
Ah yeah sounds good. I'll remove this then.